roles/big-service/vars/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+
+service_name: big-service
+script_name: big-service.py
```

Comment:
This is a great abstraction, and indeed makes the future service development much easier.

However, there is a more interesting approach (really advanced though):
Have a common role (lets call it `common_service`), with all the common deploy code.
Now, you can create a role per service, but have them depend on another role, and pass it the relevant argument.
After that, all you need to do is add a dependency (under `meta/main.yml`) with the following content (note how to pass arguments):
```
---
dependencies:
  - { role: common_service, service_name:big_service }
```

This way, less code is duplicated, and it seems much more elegant.